src/modules/partition/jobs/CreatePartitionTableJob.cpp

```diff
@@ -87,22 +87,34 @@ CreatePartitionTableJob::exec()
     }
 
     QScopedPointer< PartitionTable > table( createTable() );
-    cDebug() << "Creating new partition table of type" << table->typeName()
-             << ", uncommitted yet:\n" << table;
+    cDebug() << "Creating new partition table of type:" << table->typeName()
+             << " - Uncommitted yet:" << table;
 
     QProcess lsblk;
     lsblk.setProgram( "lsblk" );
     lsblk.setProcessChannelMode( QProcess::MergedChannels );
     lsblk.start();
     lsblk.waitForFinished();
-    cDebug() << "lsblk:\n" << lsblk.readAllStandardOutput();
+
+    QByteArray byte = lsblk.readAllStandardOutput();
+    QStringList lines = QString(byte).split(("\n"),QString::SkipEmptyParts);
+    cDebug() << "CreatePartitionTableJob asked for lsblk output:";
+
+    foreach (auto line, lines)
+        cDebug() << " " << line;
```

Comment:
We're running *lsblk* and *mount* purely for debugging-output purposes here. That seems rather expensive, really. And there's the code duplication between this block, and the processing for *mount*, below. This is only two repetitions .. if there's a third, I'd say it's time to add a nice (template) class into the Logger namespace for neatly dumping stdout from a finished QProcess.